nap/data/RoomData.js

```diff
@@ -25,14 +25,6 @@ var AppConfig = require('../config/AppConfig');
  * @param {string|Array<string>} roomNames List of room names or a single room name
```

Comment:
These comments are documentation for the followed function. They can be removed.